### PR TITLE
THE MEGASEED 10 STRANGE SEEDS VIA COIN ARE BACK BABY

### PR DESCRIPTION
**Pull request by Jujumatic**
Remember when those TG fags deleted the ability to get those 10 strange seeds by shoving a coin into the seed vendor's butt? Well this PR brings them back! Insert any coin and you can get em just like you used to be able to. I also put up the number of seeds you unlock from hacking the vendor from 2 to 3 because I think it's pretty dumb that you get less of those seeds compared to all the others. And you get 3 strange seeds right off the bat, too! You can thank me later

### CODE REVIEW

**Comment by WarRage333** [comment]
PRAISE LORD
RE-ADD two diamond coins to the slot machine then so RnD and Botany dont have to murder each other for it

**Comment by DaydreamPuppeh** [comment]
Yes please

**Comment by JohnGinnane** [comment]
Well this couldn't be more conveniently timed. Have a read of this:

https://github.com/HippieStation/HippieStation/tree/master/hippiestation#vending-machine

And take a look at this:

https://github.com/HippieStation/HippieStation/blob/master/hippiestation/code/modules/vending/autodrobe.dm

Use the new `hippie` vars to add products, contraband and/or premium items to the vending machine

**Comment by Jujumatic** [comment]
This took way too much to fix up, anyway - it's ready to get merged

File: hippiestation/code/modules/vending/megaseed.dm [hunk 1]
```diff
@@ -1,11 +1,11 @@
...
\ No newline at end of file
+	hippie_contraband = list(/obj/item/seeds/amanita = 1,
+		              /obj/item/seeds/glowshroom = 1,
+		              /obj/item/seeds/liberty = 1,
+		              /obj/item/seeds/nettle = 1,
+					  /obj/item/seeds/plump = 1,
+					  /obj/item/seeds/reishi = 1,
+					  /obj/item/seeds/cannabis = 1,
+					  /obj/item/seeds/starthistle = 1,
+					  /obj/item/seeds/random = 1) //THE 3 SEEDS ARE BACK BABY
```
Comment (WarRage333) [inline hippiestation/code/modules/vending/megaseed.dm:10]: why nerf to 1?

Comment (Jujumatic) [reply hippiestation/code/modules/vending/megaseed.dm:10]: it's not nerfing to 1, what this is doing is essentially ADDING 1 extra seed to what the tg list currently has. it's done like this so if tg changes their vendor item list there's less chance of the changes raping us

**Comment by Jujumatic** [comment]
I'm not sure what's up with all those line deletions but the map bot has shown all the changes I made. Weird af

**Comment by Jujumatic** [comment]
Shit I did it on the wrong map, 1 sec

**Comment by JohnGinnane** [comment]
did you change maps?

**Comment by ghost** [comment]
+1 yea

**Comment by Jujumatic** [comment]
Ok I'll make the map changes in a separate PR because that's clearly a lot more delicate than this. MERGE THIS SHIT ASAP BEFORE THE MAPS BREAK AGAIN AAAGGHHHH

**Review by JohnGinnane — APPROVED** [review]

**Comment by JohnGinnane** [comment]
did you test this

**Comment by JohnGinnane** [comment]
this is just a suggestion but it might be an idea to re-create this PR so it's much cleaner, without all these commits. The code is very straight forward so it shouldn't take long

**Comment by Jujumatic** [comment]
Tested and I don't think there's a need to recreate the pr cause... it'll work just the same kek

**Comment by JohnGinnane** [comment]
need one or two more votes I think

**Comment by KayeArray** [comment]
why are there **150** commits?

**Comment by Jujumatic** [comment]
i needed to update the branch lol

just look at the changed files, everything is fine

**Comment by JohnGinnane** [comment]
i suspect merging recent commits into this branch. The net effect is.. nothing but it looks a little dirty that's all

**Comment by Jujumatic** [comment]
i mean i can close it and reopen it but i don't feel like there's much of a point to it - the entire thing is ready to merge as it is but if it's for the sake of neatness then i'll do it if you really want me to?

**Comment by JohnGinnane** [comment]
Nah I think we're good, master already has those commits

**Comment by Jujumatic** [comment]
can we merge now so i can appease botany

**Comment by JohnGinnane** [comment]
I think we need head input @HippieStation/head

**Comment by Pyko1** [comment]
sure why not

**Pull request merged**